### DISCUSSION TITLE
Limitations of external editor

### DISCUSSION THREAD

**Discussion by jaroslavknotek** [qutebrowser/qutebrowser#7239] [Q&A] [answered]
Hello, 

recently, I've learnt that you can externalize writing text to web-page input box by pressing `ctrl-e` in INSERT mode. However, it doesn't work at some web pages. One particular example is jupyter notebook. It is a tool that creates web page which communicates with python kernel. To reproduce it, you just install `jupyter`, run `jupyter notebook` and create new notebook, focus empty cell to enter INSERT and then press `ctrl-e`. Unlike with other web pages (like github), saving the buffer will not make the text apper in the focuset input box. (If you had typed anything in the jupyter cell before opening the external editor, it will appear in the editor as well, so the connection from web to external editor works. The other direction does not).

Is this a bug or this approach has some limitations? I tried to look around but I wasn't very successful.

My setup is:
```
qutebrowser v2.5.0
Git commit: 64e8c7132 on qt6-v2 (2022-05-26 14:37:27 +0200)
Backend: QtWebEngine 6.3, based on Chromium 94.0.4606.126
Qt: 6.3.0
```

and in the configuration, I have this option:
```
c.editor.command = ["emacs", "+{line}:{column}" ,"{file}"]
```
Since some other web pages work, I think that the editor itself is not the problem.

**Comment by The-Compiler** [comment, marked as answer]
See #2215
